build.sbt

```diff
@@ -0,0 +1,41 @@
+import sbt.Keys.version
+
+name := {
+  println(s"[info] Welcome to scalagen ${version.value}")
```

Comment:
I recently found out `onLoadMessage` is designed for this exactly
https://github.com/scalacenter/macros/blob/5bf9c98d9c3e35ab4295e634433d962f0aef57ba/build.sbt#L4-L7

Reply:
Cool. I'll change it.